src/main/java/com/lewisd/maven/lint/plugin/CheckMojo.java

```diff
@@ -232,23 +231,20 @@ String generateErrorMessage(List<String> outputReportList)
         return message.toString();
     }
 
-    private File getOutputFileForReport(String report)
-            throws MojoExecutionException {
-        final File outputFile;
+    private File getOutputFileForReport(String report) throws MojoExecutionException {
         if ("summary".equals(report)) {
-            outputFile = summaryOutputFile;
+            return summaryOutputFile;
         } else if ("xml".equals(report)) {
-            outputFile = xmlOutputFile;
+            return xmlOutputFile;
         } else if ("html".equals(report)) {
-            outputFile = htmlOutputFile;
+            return htmlOutputFile;
         } else {
             throw new MojoExecutionException("Unsupported report: '" + report + "'");
         }
-        return outputFile;
```

Comment:
I prefer the way this was before, minimizing the number of returns from a method.  How would you feel about changing that back and tweaking the checkstyle.xml (if necessary)?